powerapi-daemon/build.sbt

```diff
@@ -0,0 +1,44 @@
+enablePlugins(JavaServerAppPackaging)
+
+name := "powerapi-daemon"
+
+libraryDependencies ++= Seq(
+  "commons-daemon" % "commons-daemon" % "1.0.15"
+)
+
+lazy val downloadBluecoveApp = taskKey[File]("download-bluecove-app")
```

Comment:
It will be more sure to change the name of task (downloadBluecoveApp and associated name)